validator/sawtooth_validator/server/cli.py

```diff
@@ -137,6 +138,26 @@ def _split_comma_append_args(arg_list):
     return new_arg_list
 
 
+def load_validator_config(first_config, config_dir):
+    default_validator_config = load_default_validator_config()
+    conf_file = os.path.join(config_dir, 'validator.toml')
+
+    toml_config = load_toml_validator_config(conf_file)
+
+    return merge_validator_config(
+        configs=[first_config, toml_config, default_validator_config])
+
+
+def create_valdiator_config(opts):
+    return ValidatorConfig(
+        bind_network=opts.network_endpoint,
```

Comment:
I am working on it now, but it will be in a separate PR because of the amount of docker file changes necessary.